### ISSUE TITLE
driver kvm: vague error message "Error getting state for host"

### ISSUE THREAD

**Issue by ericchiang** [kubernetes/minikube#937]
**Minikube version** (use `minikube version`):

```
$ minikube version
minikube version: v0.14.0
```
**Environment**:
- **OS** (e.g. from /etc/os-release): 
$ cat /etc/os-release 
NAME=Fedora
VERSION="24 (Workstation Edition)"
ID=fedora
VERSION_ID=24
PRETTY_NAME="Fedora 24 (Workstation Edition)"
ANSI_COLOR="0;34"
CPE_NAME="cpe:/o:fedoraproject:fedora:24"
HOME_URL="https://fedoraproject.org/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=24
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=24
PRIVACY_POLICY_URL=https://fedoraproject.org/wiki/Legal:PrivacyPolicy
VARIANT="Workstation Edition"
VARIANT_ID=workstation

- **VM Driver** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep DriverName`): kvm
- **Docker version** (e.g. `docker -v`):

```
$ sudo docker -v
Docker version 1.12.3, build 6b644ec
```

- **Install tools**:

Instructions provided in release and docs (curl based):

minikube: https://github.com/kubernetes/minikube/releases/tag/v0.14.0
kvm driver: https://github.com/kubernetes/minikube/blob/master/DRIVERS.md#kvm-driver

- **Others**:

**What happened**:

Ran `sudo minikube start --vm-driver kvm`, things worked. Deleted VM using `virt-manager`. Tried to run `sudo minikube-start --vm-driver kvm` got the following error message.

```
$sudo minikube start --vm-driver kvm
Starting local Kubernetes cluster...
E1219 09:26:25.726195    5969 start.go:90] Error starting host: Error getting state for host: [Code-7] [Domain-6] invalid domain pointer in virDomainGetState. Retrying.
E1219 09:26:25.726503    5969 start.go:96] Error starting host:  Error getting state for host: [Code-7] [Domain-6] invalid domain pointer in virDomainGetState
```

**What you expected to happen**:

Something more actionable. I've had networking bridge issues and those generally provided error message that explained what bridge was down or expected to be up.

**How to reproduce it** (as minimally and precisely as possible):

Don't know. I can't get minikube out of this state so I can't play around with it to try to create reproducible instructions.

**Anything else do we need to know**:

The error appears to be originating from here: https://github.com/kubernetes/minikube/blob/k8s-v1.4/pkg/minikube/cluster/cluster.go#L84

**Comment by ericchiang** [comment]
So it looks like the answer was to blow away `/root/.minikube`

```
$ sudo minikube stop
Stopping local Kubernetes cluster...
Error stopping machine:  Error stopping host: minikube: [Code-7] [Domain-6] invalid domain pointer in virDomainGetState
$ sudo rm -rf /root/.minikube/
$ sudo minikube start --vm-driver kvm
There is a newer version of minikube available (v0.14.0).  Download it here:
https://github.com/kubernetes/minikube/releases/tag/v0.14.0
To disable this notification, add WantUpdateNotification: False to the json config file at /root/.minikube/config
(you may have to create the file config.json in this folder if you have no previous configuration)
Starting local Kubernetes cluster...
E1219 09:36:49.949613    6741 start.go:90] Error starting host: Error creating host: Error creating machine: Error in driver during machine creation: [Code-55] [Domain-19] Requested operation is not valid: network 'default' is not active. Retrying.
E1219 09:36:49.949942    6741 start.go:96] Error starting host:  Error creating host: Error creating machine: Error in driver during machine creation: [Code-55] [Domain-19] Requested operation is not valid: network 'default' is not active
================================================================================
An error has occurred. Would you like to opt in to sending anonymized crash
information to minikube to help prevent future errors?
To opt out of these messages, run the command:
	minikube config set WantReportErrorPrompt false
================================================================================
Please enter your response [Y/n]: 

```

Which at least gets me back to a better error message.

**Comment by mykaul** [comment]
@ericchiang - why did you close the issue? I just had the same issue. Same workaround worked, but it's really non-intuitive.
